lib/spack/llnl/util/filesystem.py

```diff
@@ -133,7 +130,11 @@ def _get_mime_type():
     """Generate method to call `file` system command to aquire mime type
     for a specified path
     """
-    return file_command("-b", "-h", "--mime-type")
+    if is_windows:
+        # -h option (no-dereference) does not exist in Windows
+        return file_command("-b", "--mime-type")
```

Comment:
Is this always true or only for the version you installed via choco? I.e. will this be true if we have a spack derived version?

Reply:
I installed file using scoop and it also has the missing option
     -h, --no-dereference
             option causes symlinks not to be followed (on systems that sup‐
             port symbolic links).  This is the default if the environment
             variable POSIXLY_CORRECT is not defined.
I looked at the file package and it has references to MINGW libraries. So this is probably what we need to use if we want to build this in spack. I am not sure why that option is missing - maybe it has to do with how links are accessed in Unix vs Windows.

Reply:
Those options are only defined if S_IFLNK. I assume this is something set by configure which fails in Windows.